src/plugins/tts/plugin.tts.js

```diff
@@ -158,18 +158,35 @@ BookReader.prototype.initNavbar = (function (super_) {
 
       $el.find('.BRcontrols').prepend(this.refs.$BRReadAloudToolbar);
 
+      const renderVoiceOption = (language) => {
+        return language.map(voice =>
+          `<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`);
+      };
+
       const renderVoicesMenu = (voicesMenu) => {
         voicesMenu.empty();
+
+        const bookLanguages = this.ttsEngine.getVoices().filter(v => v.lang.startsWith('en')).sort((a,b) => {
+          if (a.lang > b.lang) return 1;
+          if (a.lang < b.lang) return -1;
+          return 0;
+        });
+        const otherLanguages = this.ttsEngine.getVoices().filter(v => !v.lang.startsWith('en')).sort((a, b) => {
+          if (a.lang > b.lang) return 1;
+          if (a.lang < b.lang) return -1;
+          return 0;
+        });
+
         if (this.ttsEngine.getVoices().length > 1) {
-          voicesMenu.append(this.ttsEngine.getVoices().map(
-            voice =>
-              $(`<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`)));
+          voicesMenu.append($(`<optgroup label="English (Book Language)"> ${renderVoiceOption(bookLanguages)} </optgroup>`));
```

Comment:
Let's switch this to be like so ; that way we don't have to convert `en -> English`, which is actually kind of hard in the general case!
```suggestion
          voicesMenu.append($(`<optgroup label="Book Language (en)"> ${renderVoiceOption(bookLanguages)} </optgroup>`));
```